specifyweb/specify/management/commands/run_key_migration_functions.py

```
@@ -264,12 +278,16 @@ def handle(self, *args, **options):
                             self.stdout.write(
                                 self.style.SUCCESS(f"Applying {function}...")
                             )
-                            self.funcs[function](self.stdout.write if verbose else None)
+                            run_with_retries(
+                                lambda: self.funcs[function](self.stdout.write if verbose else None)
```

Ruff (0.15.17)

[warning] Function definition does not bind loop variable `function`

(B023)